### DISCUSSION TITLE
How to get the charge on each Amino residue?

### DISCUSSION THREAD

**Discussion by samit1194** [jensengroup/propka#175] [Q&A]
Hi,

Once pKa File is generated. It has the following columns to see the charges.

1. **Sidechain Hydrogen Bond**: This column indicates the hydrogen bonding interactions involving the sidechain of the residue.

2. **Backbone Hydrogen Bond**: This column indicates the hydrogen bonding interactions involving the backbone of the residue.

3. **Coulombic Interaction:** This column provides information about the coulombic interactions of the residue.

To assess the protonation states, I can compare the pKa value with the pH of the solvent, along with the respective charge on the Amino residue.

Since I will be using the AMBER FF, ASP would need to be renamed as ASH, and likewise for other Amino acids.   

Which columns must I consider for calculating the charge on the Amino residue? and what is the logic behind it?

**Comment by YZXyinzhaoxu** [comment]
Excuse me, I was wondering where “python-m propka” will be used after downloading the propka python module? Use it in python/ jupyter notebook, or in windows power shell？

**Comment by sobolevnrm** [comment]
> Excuse me, I was wondering where “python-m propka” will be used after downloading the propka python module? Use it in python/ jupyter notebook, or in windows power shell？

This question is answered in #174 .

**Comment by sobolevnrm** [comment]
> Which columns must I consider for calculating the charge on the Amino residue? and what is the logic behind it?

It is probably easier to extract this information from a PQR file generated by [PDB2PQR](https://github.com/Electrostatics/pdb2pqr).  PDB2PQR uses PROPKA as its backend for predicting the pKa and titration state but then assigns charges based on that information.
